docs/topics/jvm/java-to-kotlin-interop.md

```diff
@@ -325,6 +325,7 @@ public class BB8 implements Robot {
     }
 }
 ```
+
 > Prior to Kotlin 1.4, to generate default methods, you could use the `@JvmDefault` annotation on these methods.
 > Compiling with `-Xjvm-default=all` in 1.4+ generally works as if you annotated all non-abstract methods of interfaces
 > with `@JvmDefault`and compiled with `-Xjvm-default=enable`. However, there are cases when their behavior differs.
```

Comment:
Can't understand should we now stop reference to the https://blog.jetbrains.com/kotlin/2020/07/kotlin-1-4-m3-generating-default-methods-in-interfaces/ blog post here (the hidden lines below) or not.

Reply:
Looks it's ok

Reply:
For me it was hard to understand whether advices from this post are still topical. But seems like they are.